app/views/widgets/home_page/_hero_panel.html

```diff
@@ -1,44 +1,28 @@
 <div id="heroWrapper">
```

Comment:
The HTML in this file was outdated, which was making it hard to test my updates, and it's supposed to mimic the live site. So now it matches what's in production right now. Except it's just a static image instead of the video background.